model/logs/mobile-crashes.yaml

```
@@ -0,0 +1,42 @@
+groups:
+  - id: mobile-crash-event
+    type: event
+    brief: >
+      This document defines attributes for crashes represented using Log Record Events.
+      `event.domain` is 'device' and `event.name` is 'crash
+    attributes:
+      - id: platform
+        requirement_level: "required"
+        brief: >
+            This attribute the platform type of the device on which the crash event occurred.
+        type:
+          allow_custom_values: false
+          members:
+            - id: android
+              value: 'android'
+              brief: 'Value if crash happened on android device'
+            - id: ios
+              value: 'ios'
+              brief: 'Value if crash happened on ios device'
+      - id: isappinforeground
+        requirement_level: "required"
+        brief: >
+          This attribute indicates whether the app was in the foreground or not during the crash.
+        type:
+          allow_custom_values: false
+          members:
+            - id: true
+              value: 'true'
+              brief: 'Value if crash happened in foreground'
+            - id: false
+              value: 'false'
+              brief: 'Value if crash did not happen in foreground'
```

Wait, can't this just be boolean?
```suggestion
        type: boolean
```